app/src/organisms/Devices/ProtocolRun/SetupLiquids/SetupLiquidsList.tsx

```diff
@@ -303,14 +302,14 @@ export const LiquidsListItemDetails = (
     description,
   } = props
   return (
-    <Flex flexDirection={DIRECTION_ROW}>
+    <Flex flexDirection={DIRECTION_ROW} alignItems={ALIGN_CENTER}>
       <Flex
         css={LIQUID_BORDER_STYLE}
-        padding={SPACING.spacing12}
+        padding="0.5rem"
```

Comment:
Could you ask about [this](https://www.figma.com/design/MlZjOO8aZXsw3HfTBoyw9U?node-id=447-8054&m=dev#858943894) to Mel?

```suggestion
        padding={spacing8}
```

Reply:
Yes, I will make a note to followup with Mel if she requires .55rem rather than .5

Reply:
that sounds reasonable!
thanks!